app/lib/smaxtec_api.rb

```diff
@@ -0,0 +1,67 @@
+class SmaxtecApi < ApplicationController
```

Comment:
just remove `< ApplicationController` I don't see any controller specific methods, so this should still work

Apart from that this PR looks ready to merge